deploy-manage/users-roles/cluster-or-deployment-auth/user-authentication.md

```diff
@@ -38,4 +43,37 @@ $$$oidc$$$
 
 $$$saml$$$
 
-$$$token-authentication$$$
\ No newline at end of file
+$$$token-authentication$$$
+
+
+
+Review the following topics to learn about authentication in your Elasticsearch cluster:
+
+### Set up user authentication
+
+* Learn about the available [realms](/deploy-manage/users-roles/cluster-or-deployment-auth/authentication-realms.md) that you can use to authenticate users
+* Manage passwords for [default users](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-users.md)
```

Comment:
```suggestion
* Manage passwords for [built-in users](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-users.md)
```

Same as before, but feel free to make your choice.
Take in mind that for roles you are still using `built-in` roles.